gadgets/mri_core/AugmentImageMetadataGadget.cpp

```diff
@@ -0,0 +1,33 @@
+#include "AugmentImageMetadataGadget.h"
+
+#include "log.h"
+
+Gadgetron::AugmentImageMetadataGadget::AugmentImageMetadataGadget(const Core::Context& context, const Core::GadgetProperties& props)
+    : PureGadget(context,props)
+{}
+
+Gadgetron::Core::Image<std::complex<float>> Gadgetron::AugmentImageMetadataGadget::process_function(
```

Comment:
The sparsity of the augment function ( and the naming approach) leaves a bit to be desired. If we can add some comments about what this is augmenting specifically, It may make things a bit easier in the future.

Reply:
I meant it as an introduction of a pattern where we can patch up what are essentially deficiencies in ISMRMRD (like no clear definition of row and column direction). I have patched the code so that we don't update if they are already set (e.g. in a recon that flips the image). I suspect there will be other fields added later (see https://ismrmrd.readthedocs.io/en/latest/mrd_image_data.html#metaattributes).